data-prepper-plugins/cloudwatch-logs/build.gradle

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id 'java'
+    id 'java-library'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    api project(':data-prepper-api')
+    implementation project(':data-prepper-plugins:aws-plugin-api')
+    testImplementation platform('org.junit:junit-bom:5.9.1')
+    implementation platform('software.amazon.awssdk:bom:2.20.56')
```

Comment:
You don't have to define bom dependency here, I think it's configured in root build.gradle. If you have to use latest version update it [here](https://github.com/opensearch-project/data-prepper/blob/ff2b9dfacc42adbd08e16dbda7bc09ec9d6d4b1f/build.gradle#L199).

Reply:
I have removed the sdk bom dependecy for the next revision.